tunder/lib/src/console/commands/migrations/make_migration_command.dart

```diff
@@ -34,16 +41,18 @@ class MakeMigrationCommand extends Command {
 
     if (logging != null)
       logging.complete('Migration created: ${createdFile.path}');
```

Comment:
The conditional member access operator may simplify this expression. The `complete` method will be called only if the `logging` isn't null.
```dart
logging?.complete('Migration created: ${createdFile.path}');
```

Reply:
Applied. Thank you for this suggestion, it's much better and avoids `if` statements. 👍🏻